### ISSUE TITLE
CMake Error at CMakeLists include could not find load file (cmake/Conan.cmake cmake/CompilerWarnings.cmake)

### ISSUE THREAD

**Issue by cp4r3z** [jabaa/generator-wombytes-cpp#9]
I'm seeing a CMake error after generating the project and attempting to build it.

```
[cmake] CMake Error at CMakeLists.txt:5 (include):
[cmake]   include could not find load file:
[cmake] 
[cmake]     cmake/Conan.cmake
[cmake] 
[cmake] 
[cmake] CMake Error at CMakeLists.txt:6 (include):
[cmake]   include could not find load file:
[cmake] 
[cmake]     cmake/CompilerWarnings.cmake
[cmake] 
[cmake] 
```

Environment: VS Code 1.51.1 on Windows 10 Version 1909 Build 18363 running Docker Desktop 2.3.0.5. Tested on Hyper-V and WSL2

**Comment by jabaa** [comment]
@cp4r3z  Thank you very much for this information. I found the problem and will fix it in the next version.

**Comment by jabaa** [comment]
Fixed in v0.7.0
